### PR TITLE
PRIME-2706 Mismatch Missing Business License flag in Admin

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

**Comment by neophyte57** [comment]
@bergomi02  I'm trying to understand what this PR fixes ... previously there was a bug that enabled CP Site Registrations to be created with No Business Licence but a Business Licence Expiry Date that was set?  Is there an instance of such data in the `dev` environment?

**Comment by bergomi02** [comment]
> @bergomi02 I'm trying to understand what this PR fixes ... previously there was a bug that enabled CP Site Registrations to be created with No Business Licence but a Business Licence Expiry Date that was set? Is there an instance of such data in the `dev` environment?

This should fix any discrepancy between the two pages. Here is the example in DEV:
image
For "Site C", the missing business license is marked "Yes".
image
in the overview page, it marked as "No".

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.ts [hunk 1]
```diff
@@ -108,6 +108,22 @@ export class SiteRegistrationTableComponent implements OnInit, AfterViewInit {
     }
   }
 
+  public displayMissingBusinessLicence(row: SiteRegistrationListViewModel) {
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.ts:111]: Please add return type

**Review by neophyte57 — APPROVED** [review]
Looks good👍

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.ts (hunk 1 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.ts]: return type added.

**Pull request merged**